shard.override.yml

```diff
@@ -3,4 +3,6 @@ dependencies:
     github: luckyframework/teeplate
   shell-table:
     github: jwaldrip/shell-table.cr
-    branch: master
\ No newline at end of file
+    branch: master
+  inflector:
+    github: anykeyh/inflector.cr
```

Comment:
Any reason not to use a fork instead of the upstream source? (https://github.com/phoffer/inflector.cr)
I guess this was because of the crystal 1.0.0 compatibility but it seems like upstream has been updated.